src/olympia/files/migrations/0019_rename_sitepermission_filesitepermission.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.9 on 2021-12-02 11:09
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('files', '0018_auto_20211130_1539'),
+    ]
+
+    operations = [
+        migrations.RenameModel(
+            old_name='SitePermission',
+            new_name='FileSitePermission',
+        ),
+    ]
```

Comment:
just squash these two migration together, imo (apologies to your local db migration state 😛 )